addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -194,10 +194,14 @@ var ImageWidget = MediaWidget.extend({
         'click .existing-attachments [data-src]': '_onImageClick',
         'dblclick .existing-attachments [data-src]': '_onImageDblClick',
         'click .o_existing_attachment_remove': '_onRemoveClick',
+        'click .o_load_more_images': '_onLoadMore',
+        'keyup #imageurl': '_onImageURL',
```

Comment:
`_onImageURL` does not mean anything to me, `ImageURL` is not a verb/event. Please use meaningful names, here I suppose it is an input, so `_onImageURLInput` and the `event` would not be `keyup` but `input`.

Reply:
Already modifed but I agree on meaningful names